portfolio/src/main/webapp/script.js

```diff
@@ -37,8 +37,9 @@ function getComments() {
     const commentList = document.createElement('dl');
     document.getElementById("display-comments").innerHTML = '';
     document.getElementById("display-comments").appendChild(commentList);
-
+    console.log(jsonCommentMap);
```

Comment:
Do you still want to log in this code, or was it just for debugging?

Reply:
Just debugging, I'm removing it.